index af1c352..24ad4a7 100644
--- a/framework/src/main/resources/master.sh
+++ b/framework/src/main/resources/master.sh
@@ -121,9 +121,9 @@ if ! [ "x${MASTER}" = "x" ] ; then
 fi
 
 #enable	remote JMX
-D_VARS="${D_VARS} -Dcom.sun.management.jmxremote.port=$JMX_MASTER_PORT"
-D_VARS="${D_VARS} -Dcom.sun.management.jmxremote.authenticate=false"
-D_VARS="${D_VARS} -Dcom.sun.management.jmxremote.ssl=false"
+#D_VARS="${D_VARS} -Dcom.sun.management.jmxremote.port=$JMX_MASTER_PORT"
+#D_VARS="${D_VARS} -Dcom.sun.management.jmxremote.authenticate=false"
+#D_VARS="${D_VARS} -Dcom.sun.management.jmxremote.ssl=false"
 
 java ${JVM_OPTS} -classpath $CP ${D_VARS} $SLAVE_COUNT_ARG $SLAVE_COUNT_ARG_I org.radargun.LaunchMaster -config ${CONFIG} > stdout_master.out 2>&1 &
 export RADARGUN_MASTER_PID=$!
